common/client-libs/gateway-client/src/socket_state.rs

```diff
@@ -37,9 +40,22 @@ type WsConn = JSWebsocket;
 
 type SplitStreamReceiver = oneshot::Receiver<Result<SplitStream<WsConn>, GatewayClientError>>;
 
+pub(crate) fn ws_fd(_conn: &WsConn) -> Option<RawFd> {
```

Comment:
_conn -> conn

Reply:
maybe we should just target lock everything to do with file descriptors? It's not like we could even use them in wasm

Reply:
Yes, in wasm they would not appear. But I think it's a bit better to use `Option` instead, and just leave the wasm macros to as little code as possible, instead of applying it all the way up